deployment/k8sPaiLibrary/maintainlib/kubectl_conf_check.py

```diff
@@ -59,16 +59,22 @@ def check(self):
 
         try:
             local_kubectl_conf = common.load_yaml_file("{0}/config".format(self.kube_conf_path))
-            api_server_address = local_kubectl_conf['clusters'][0]['cluster']['server']
-
             api_server_address_pai_conf = "http://{0}:8080".format(self.cluster_config['clusterinfo']['api-servers-ip'])
+	    api_server_address_aks_conf = "https://{0}:443".format(self.cluster_config['clusterinfo']['api-servers-ip'])
```

Comment:
8080 and 443 are just ad hoc config info that should go to cluster config file

Reply:
move it as api-server-port config at pai k8s config file